PyBit Minecraft Bot - Exception Dump('deposit', "Error: Can't find cobblestone in slots [54 - 90], (item id: 35)\n    at transferOne (/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/javascript/js/node_modules/mineflayer/lib/plugins/inventory.js:294:32)\n    at transfer (/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/javascript/js/node_modules/mineflayer/lib/plugins/inventory.js:280:11)\n    at window.deposit (/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/javascript/js/node_modules/mineflayer/lib/plugins/inventory.js:377:13)\n    at Bridge.call (/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/javascript/js/bridge.js:136:42)\n    at Bridge.onMessage (/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/javascript/js/bridge.js:231:25)\n    at Socket.<anonymous> (/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/javascript/js/bridge.js:292:18)\n    at Socket.emit (node:events:507:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)")